### ISSUE TITLE
start with steamos?

### ISSUE THREAD

**Issue by AireagoirSteam** [LukeShortCloud/winesapOS#602]
Im having issues booting arch with steamos on it never seemed to work since my main machine would boot the first option with linux lts. thwn when picking the steamos option would start take a while complain about the fan manager daemon and just not do anything for a while.

on my second machine picking the steamos option starts linux with lts successfully and when the steamos option complains about not being able to start light display manager and then complains about the fan manager daemon and just does nothing after waiting 10 ti 15 minutes.

**Comment by LukeShortCloud** [comment]
Hey @AireagoirSteam , thanks for reporting this issue! What are the specs of your computer?

The first thing I recommend trying is to update the graphics drivers. There is a known issue with those in winesapOS 3.2.1. On your second computer, boot into Linux LTS, open `Konsole`, and then run these two commands:

```
sudo pacman -S -y -y
sudo pacman -S mesa-steamos mesa-vdpau-steamos opencl-mesa-steamos vulkan-intel-steamos vulkan-mesa-layers-steamos vulkan-radeon-steamos vulkan-swrast-steamos lib32-libva-mesa-driver-steamos lib32-mesa-steamos lib32-mesa-vdpau-steamos lib32-opencl-mesa-steamos lib32-vulkan-intel-steamos lib32-vulkan-mesa-layers-steamos lib32-vulkan-radeon-steamos lib32-vulkan-swrast-steamos libva-mesa-driver-steamos
```

The second thing I would try, if the first thing doesn't work, is to use winesapOS 3.2.0 and see if that works. At some point, you'll want to run the "winesapOS Upgrade" desktop shortcut. The upgrade process already addresses a few issues in winesapOS 3.2.1.

Other things to note:
- Linux LTS is preferred. Linux SteamOS is old and most functionality (except for HDR) has been emerged upstream already. Once HDR support is merged upstream, I may even remove the Linux SteamOS kernel because it's very old, insecure, and specific to the Steam Deck. I'm not surprised that  Linux SteamOS doesn't work for your two computers.
- The fan manager is a red herring. You can ignore that as it's not related to any boot or hardware issues. I've addressed that in our upcoming winesapOS 3.3.0 release.

Hope this helps! Let me know how it goes.

**Comment by LukeShortCloud** [comment]
@AireagoirSteam winesapOS 3.3.0 has been released to address the issues you were encountering!

https://github.com/LukeShortCloud/winesapOS/releases/tag/3.3.0
